src/LightPanels/LightPanels.less

```diff
@@ -95,6 +95,7 @@
 
 		> .offscreen {
 			visibility: hidden;
+			opacity: 0;
```

Comment:
I thought that panel will be destroyed after off-screened, do we still need to have visibility: hidden prop?